client/lib/post-normalizer/rule-content-make-images-safe.js

```diff
@@ -31,7 +31,7 @@ const removeUnwantedAttributes = ( node ) => {
  */
 const imageShouldBeRemovedFromContent = ( imageUrl ) => {
 	if ( ! imageUrl ) {
-		return;
+		return false;
```

Comment:
This is a safe change, since it's used only once in an `if()` and the result is implicitly converted to a boolean anyway.